README.md

```diff
@@ -8,29 +8,31 @@ This library builds on top of the [Microsoft.OpenAPI.NET](https://github.com/mic
 
 ## Getting started
 
+### Generating SDKs
+
+1. Install the required tools for your language. (refer to the [Supported Languages](#supported-languages) table under the **Required tools & dependencies** column)
+1. Intialize the target project and add the required dependencies. (refer to the [Supported Languages](#supported-languages) table under the **Required tools & dependencies** column)
```

Comment:
This took me a while to grok what you ment here and that was only after I[ stumbled across some docs that elaborated on the "Initialize the the target project"](https://github.com/microsoft/kiota/blob/main/docs/requiredtools/dotnet.md#initializing-target-projects).

Like, what does that even mean?

Trying to understand that page (dotnet.md) is really really really confusing. I honestly have no idea 

- what's it's goal really is
- what i'm trying to do

also, that "creating an application registration" was a stomach punch :( If I was lost, before this section ... I was now crying at this point.

Reply:
Let's unpack this very valuable feedback here:
- What does "initialize the project" mean? We want to make sure people add the right dependencies in their target project, and maybe do a few extra steps if they are talking to Microsoft Graph/an API behind AAD. => maybe we can reword that part?
- dotnet page (and others): maybe we can reword things a little to make it easier to understand, don't hesitate to throw in suggestions here. I'll try to make some additional changes.
- initialize application: already added a note to instruct people to skip this if it doesn't apply to them.

Reply:
```suggestion
1. Add the required dependencies. (refer to the [Supported Languages](#supported-languages) table under the **Required tools & dependencies** column)
```

What do you think of that?

Reply:
Yep - better! I still feel like I need to see the entire readme etc .. for context. but yes, this is better IMO.